src/Nexucis/Elasticsearch/Helper/Nodowntime/IndexHelper.php

```diff
@@ -457,7 +458,11 @@ public function searchDocuments($alias, $query = null, $type = null, $from = 0,
         );
 
         if (is_array($query)) {
-            $params['body'] = array('query' => $query);
+            $params['body']['query'] = $query;
+        }
+
+        if (is_array($source)) {
+            $params['body']['_source'] = $source;
```

Comment:
as you can see on the [php documentation](https://github.com/elastic/elasticsearch-php/blob/master/src/Elasticsearch/Client.php#L921) , the `_source` parameter is not something that shall be in the body but directly in the root of the parameter  : 

```php
$params['_source'] = $source
```

Reply:
Hey. Yes, it's true. I miss that because it works in both ways.